src/Evolution/Systems/GrMhd/ValenciaDivClean/System.hpp

```diff
@@ -21,14 +21,18 @@
 /// \ingroup EvolutionSystemsGroup
 /// \brief Items related to general relativistic magnetohydrodynamics (GRMHD)
 namespace grmhd {
-/// The Valencia formulation of ideal GRMHD with divergence cleaning.
+/// The Valencia formulation of ideal GRMHD with divergence cleaning coupled
+/// with electron fraction.
 ///
 /// References:
 /// - [Numerical 3+1 General Relativistic Magnetohydrodynamics: A Local
 /// Characteristic Approach](http://iopscience.iop.org/article/10.1086/498238)
 /// - [GRHydro: a new open-source general-relativistic magnetohydrodynamics code
 /// for the Einstein toolkit]
 /// (http://iopscience.iop.org/article/10.1088/0264-9381/31/1/015005)
+///
+/// FIXME : add a reference on Y_e?
```

Comment:
I need a help on this. Should we add any reference on Y_e or just leave it as now?

Reply:
I'm not sure what people's preferences are as there are many references on this, most of which then focus more of the neutrino transport part:

How about these? (They don't do M1 or any sophisticated transport, since we only need to reference the electron fraction equation, but any other paper by Francois etc. would also do)

https://articles.adsabs.harvard.edu/pdf/1996A%26A...311..532R
https://arxiv.org/pdf/1304.3384.pdf

Reply:
The second one has the GR version of equation so I'm good with that. Once we have a paper with neutrino treatment in the future maybe we can add that here as well.

Reply:
Sounds good, then let's take that one for now!